Update layoute_rule and trigger schemas to handle complex structure (#3833)
demisto/demisto-sdk: CHANGELOG.md

## Unreleased
* Added support to automatically detect the correct file model by file path when reading files.
* Fixed an issue where **run-playbook** command didn't work if the *url* argument was not provided.

> * Fixed an issue in **validate** command, where layout_rules and trigger schemas will now handle complex structures.

Reviewer: ```suggestion
* Fixed an issue where **validate** command failed on valid complex layout rules and triggers.
```
